Framework/Algorithms/src/RemoveLowResTOF.cpp

```diff
@@ -265,9 +259,10 @@ double RemoveLowResTOF::calcTofMin(const std::size_t workspaceIndex) {
   double tmin = 0.;
   if (isEmpty(m_wavelengthMin)) {
     std::map<detid_t, double> offsets; // just an empty offsets map
+    Geometry::Instrument_const_sptr instrument = m_inputWS->getInstrument();
     double dspmap =
-        Instrument::calcConversion(m_L1, beamline, beamline_norm, samplePos,
-                                   m_instrument, detNumbers, offsets);
+        Instrument::calcConversion(l1, beamline, beamline_norm, samplePos,
+                                   instrument, detNumbers, offsets);
```

Comment:
This function is a bit of a mess and we should probably clean this up, making the code here nicer as well, but that is beyond the scope of this PR.